meerkat_frontend/src/js/technical/misc.js

```diff
@@ -585,9 +586,9 @@ function exportTableToXLS(tableID, filename) {
     });
 
     //Return the percentage values to the HTML design ...
-    for (var j = 0; j <= chartPercentageList.length - 1; j++) {
-        chartPercentageList[j].innerHTML = oldValueArray[j];
-    }
+    chartPercentageList.each(
+        function(index, element){$(element).html(oldValueArray[index]);}
```

Comment:
Even though it's a single command operation I'd break lines here:
```
chartPercentageList.each(
     function(index, element){
          $(element).html(oldValueArray[index];
 })

Reply:
Yeah fine.  I agree with this.